packages/discord.js/src/util/Options.js

```diff
@@ -35,7 +35,7 @@ const { version } = require('../../package.json');
  * @property {IntentsResolvable} intents Intents to enable for this connection
  * @property {number} [waitGuildTimeout=15_000] Time in milliseconds that clients with the
  * {@link GatewayIntentBits.Guilds} gateway intent should wait for missing guilds to be received before being ready.
- * @property {SweeperOptions} [sweepers={}] Options for cache sweeping
+ * @property {SweeperOptions} [sweepers={threads: {interval: 3600, lifetime: 14400}}] Options for cache sweeping
```

Comment:
```suggestion
 * @property {SweeperOptions} [sweepers=DefaultSweeperSettings] Options for cache sweeping=
```

the shortest way

Reply:
Sorry.

Thank you for the suggestion.
I will modify it to your form.

Reply:
Btw you can just commit suggestion on github site

Reply:
I've modified it by adding an additional 'this.' What do you think?

I used the following code and others as reference
https://github.com/discordjs/discord.js/blob/main/packages/discord.js/src/sharding/ShardingManager.js#L177

Sorry if this is wrong.

Reply:
> the shortest way, but i am not too sure if its right

If that links properly to what it returns in the docs, then it's alright.
If not I think the original way is appropiate.

Reply:
Nah docs shows just `default` so lnks wony work there

Reply:
But this long syntax looks horrible in docs